docs/rules/filename-case.md

```diff
@@ -57,3 +65,24 @@ Or set the `cases` option to allow multiple cases:
 	}
 ]
 ```
+
+### ignore
+
+Type: `RegExp[]`\
+Default: `[]`
+
+Filenames to ignore.
+
```

Comment:
Could be useful to mention some real-world use-cases for this option, like done in https://github.com/sindresorhus/eslint-plugin-unicorn/issues/395